tests/integration/test_logger_integration.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Integration tests for the logging mechanism used in Strawberry Fields."""
```

Comment:
Note: this should actually go in the `frontend/` folder. The `integration` test folder is a bit of a misnomer, and isn't for true integration tests. Instead, its for end-to-end tests which involve both the SF frontend _and_ backends:

* `tests/frontend` ("not backend"): for all unit tests and integration tests involving _just_ frontend components.

* `tests/backend` ("only backend"): for all unit tests and integration tests involving _just_ the backend components (i.e. the `strawberryfields/backend` folder)

* `tests/integration` (backend and frontend): for end-to-end tests involving both frontend and backend components.

The reason for this is historical -- at some point, we were considering moving the backends into their own repositories, hence the need to organize the tests into 'not backend', 'backend', and 'end-to-end'!

---

In this case, it is probably fine to move the  class `TestLoggerIntegration` to within the existing `test_logger.py` class 🙂 

We should also consider renaming the `integration` folder to `end_to_end` (or something similar), _or_ building out the developer documentation to make this more clear.